app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -256,6 +265,10 @@ class BrowserFragment : Fragment(), BackHandler {
         promptsFeature.withFeature { it.onActivityResult(requestCode, resultCode, data) }
     }
 
+    override fun onTrimMemory(level: Int) {
+        requireComponents.core.sessionManager.onLowMemory()
+    }
```

Comment:
This looks like something that should happen at the "app level" and not be buried somewhere in a Fragment. [Application](https://developer.android.com/reference/android/app/Application) already implements `ComponentCallbacks2` - so you can just override `onTrimMemory` there. :)

Reply:
👍

Reply:
https://github.com/mozilla-mobile/fenix/pull/1018